components/eam/bld/configure

```diff
@@ -2623,13 +2617,7 @@ sub write_filepath
     print $fh "$camsrcdir/eam/src/chemistry/utils\n";
 
     # Add source code directories for selected radiation package
-    # Note: need to add rad code even for rad none because source code in CAM
-    # physics folder needs constants defined in radconstants.F90. A better
-    # approach would be to separate components/eam/src/physics/cam into
-    # something like physics/share, physics/eam, and physics/ideal, and then
-    # only include the modules actually needed by the code. This would remove
-    # the need to build a rad package when no rad package is used.
-    if ($rad eq 'rrtmg' or $rad eq 'none') {
```

Comment:
Did you mean to remove this? I think this will break idealized compsets.

Reply:
Replaced in most recent commit.